config/initializers/1_fast_gettext.rb

```diff
@@ -1,6 +1,10 @@
 require 'fast_gettext'
 require 'gettext_i18n_rails'
 
+require "#{Rails.root}/lib/foreman/gettext/all_domains"
+require "#{Rails.root}/lib/foreman/gettext/debug"
+require "#{Rails.root}/lib/foreman/gettext/support"
```

Comment:
can't we use the fact, that lib should be in the require path and make it bit more straight forward:
```suggestion
require 'foreman/gettext/all_domains'
require 'foreman/gettext/debug'
require 'foreman/gettext/support'
```
if that is not feasible, we should use [Pathname#join](https://ruby-doc.org/stdlib-1.9.3/libdoc/pathname/rdoc/Pathname.html#method-i-join)
`Rails.root.join('lib', 'foreman', ...)`